aspnetcore/fundamentals/logging/index.md

```diff
@@ -408,9 +408,9 @@ The preceding code displays console logs when the category contains `Controller`
 
 Generally, log levels should be specified in configuration and not code.
 
-## ASP.NET Core and EF Core categories
+## ASP.NET Core categories
 
-The following table contains some categories used by ASP.NET Core and Entity Framework Core, with notes about the logs:
+The following table contains some categories used by ASP.NET Core:
```

Comment:
Per the style manual.

**Lines 243, 538, 547, and 558 should also get the same change with some wording updates to say 'in the following table' or similar.**

```suggestion
The following table contains some categories used by ASP.NET Core.
```